crates/swift-bridge-ir/src/codegen/generate_rust_tokens/shared_struct.rs

```diff
@@ -90,7 +90,15 @@ impl SwiftBridgeModule {
             }
         };
 
+        let automatic_derives;
```

Comment:
We're moving away from automatic derives, so `derives` is fine.

Reply:
[ead0af9](https://github.com/chinedufn/swift-bridge/pull/198/commits/ead0af99013d5cafb5e2cb8d0da6ba7a191d8211)